sabelotodo/tests/test_routes.py

```diff
@@ -9,7 +9,7 @@
 
 VALID_ITEM_DATA = [{'name': 'minimal', 'order': 0, 'done': False},
                    {'name': 'has_start_date', 'order': 1, 'done': False,
-                       'start_date': 'Sun, 06 Nov 94 08:49:37 -0000'},
+                       'start_date': 'Sun, 06 Nov 1994 08:49:37 -0000'},
```

Comment:
Consider using a python-generated date or double check what the date string is that you will actually receive so that this test is accurate.